README.md

```diff
@@ -39,7 +40,7 @@ Just create udev rule for hidraw device with autorun set defaults (/etc/udev/rul
 KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="2573", ATTRS{idProduct}=="0017", GROUP="plugdev", MODE="0660", RUN+="/usr/local/bin/maya22-control -d"
 ```
 
-GUI application
+2024-09-91 - New variable to disable headphone
+
+New GUI application in Construction  
 ---------------
-You can find GUI more useful for your needs, so please check the following project:
```

Comment:
That's nice that you developing a new gui app, but why to remove a link to the old gui from Psy-Kai? Let's leave it and add your in the list when it will be ready?